modules/server/src/topics/timetopic.cpp

```diff
@@ -94,6 +99,37 @@ void TimeTopic::handleJson(const nlohmann::json& json) {
             sendFullTimeData();
         }
     });
+
+    _deltaTimeStepsCallbackHandle = global::timeManager.addDeltaTimeStepsChangeCallback(
+        [this]() {
+            const std::vector<double> steps = global::timeManager.deltaTimeSteps();
```

Comment:
I'd say either `std::vector<double>` or `const std::vector<double>&`  If using the first, there will be no copy since the optimizer will get rid of it probably, the second is a bit more verbose and makes it clear that it's constant